### ISSUE TITLE
Doesn't build on OpenBSD

### ISSUE THREAD

**Issue by killruana** [twpayne/go-shell#6]
Hi,

I'm trying to build chezmoi v1.8.3 on OpenBSD 6.7 and I have the following error:

```
$ make generate
go generate
# github.com/twpayne/go-shell
../../go/pkg/mod/github.com/twpayne/go-shell@v0.2.0/shell_posix.go:30:18: undefined: cgoGetUserShell
main.go:4: running "go": exit status 2
*** Error 1 in /home/killruana/tmp/chezmoi (Makefile:8 'generate')
```

**Comment by twpayne** [comment]
Thanks for reporting this. This should be fixed by 8d6628b9b83c49e54f3bdfdb7c6caa9390d2f5c7.

**Comment by killruana** [comment]
Build fine, thx
